src/reference/asciidoc/whats-new.adoc

```diff
@@ -14,11 +14,16 @@ Please submit GitHub issues and/or pull requests for additional entries in that
 
 This version requires the 2.2.0 `kafka-clients` or higher.
 
-==== Producer Factory Changes
+==== Producer and Consumer Factory Changes
 
 The `DefaultKafkaProducerFactory` can now be configured to create a producer per thread.
 See <<producer-factory>> for more information.
 
+You can also provide `Serializer` suppliers in the constructor as an alternative to either configured classes (which require no-arg constructors)
+or constructing with `Serializer` instances, which are then shared between all Producers.
+
+The same option is also available with `Deserializer` suppliers in `DefaultKafkaConsumerFactory` - see <<kafka-container>>
```

Comment:
Missed period in the end.

Reply:
The period is still missed 😄

Reply:
Can you say like this  "`Supplier<Deserializer>` isntances" ?
It is really what we expect from end-user to provide, so why don't get a generics syntax benefit in the docs as well? 😉